src/test/moment/is_valid.js

```diff
@@ -122,12 +122,14 @@ test('valid string iso 8601 + timezone', function (assert) {
         '2010-01-30T23:59:59.999+00:00',
         '2010-01-30T23:59:59.999-07:00',
         '2010-01-30T00:00:00.000+07:00',
-        '2010-01-30T00:00:00.000+07'
```

Comment:
If we don't support it, why was there a unit test case? I had to remove it because it was failing when strict mode was enabled, so it seemed like a mistake in the tests.